src/pages/Cart/Cart.tsx

```diff
@@ -13,14 +13,18 @@ import type { Carts } from '@/types/api/Carts.type';
 import styles from './Cart.module.scss';
 
 const CartPage = () => {
+  const navigate = useNavigate();
   const cartRes = useGetApi('/carts');
   const userRes = useGetApi('/users/me');
   const { mutate: createOrder, isPending } = usePostApi('/orders');
```

Comment:
```suggestion
  const { mutate: createOrder } = usePostApi('/orders');
```
isPending이 사용되지 않는 것 같습니다!